### DISCUSSION TITLE
Packet Sender for iOS

### DISCUSSION THREAD

**Discussion by dannagle** [dannagle/PacketSender#228] [General]
Just in time for Spring (in the U.S.)! 

I am happy to announce the release of the from-scratch rewrite of Packet Sender for iOS! It was written in native tools (Xcode, SwiftUI) to hopefully be a far better experience than the old Xamarin version. 

Apple App Store:
https://apps.apple.com/app/id1558236648#?platform=iphone

Enjoy!

**Comment by robbel** [comment]
Thank you @dannagle. Could you kindly explain the Settings tab for the iPhone app, especially the two "5005" entries with a wireless symbol next to them? Does the iPhone app support binding to an ipv6 address? Thanks!

**Reply by dannagle** [reply]
Those are the UDP and TCP listen ports. It does a general port bind that should support IPv6.

**Reply by dannagle** [reply]
The other is Packet Sender Cloud login info. It lets you easily transfer packets from the desktop app to the mobile app via the free PS Cloud service. Example: https://cloud.packetsender.com/dannagle/piplayer

**Comment by postprivee** [comment]
'not available in your country' warning although it was available 2-3 weeks ago. appstore bug?

**Comment by dannagle** [comment]
The iOS app has been removed from the store for now. The Apple App store moves faster than I am currently able to keep up. I plan to fix the problems with the iOS app and put it back up eventually, but I am not sure when. I don't want to put it back up until I am confident I have the resources and a good workflow to properly support it. My goal is June.

**Comment by dannagle** [comment]
It took 2 years later, but the iPhone app is back in the store. Hopefully, things are clearer and work better now.

Thanks to everybody supporting this project. For now, I should have the resources to sustain this app.
